docs/Project-Configuration/Project-and-Distro-Settings.md

```diff
@@ -203,6 +203,11 @@ For security reasons, commits by users outside of your organization will
 not automatically be run. A patch will still be created and must be
 manually authorized to run by a logged-in user.
 
+#### Limiting when PR patches will run
+You can optionally specify the oldest commit SHA that is allowed to be a merge base
+for a PR, otherwise a PR patch will not be created. To do so, input the oldest SHA
+on your project's branch you want to accept as the merge base via the 'Oldest Allowed Merge Base' field.
```

Comment:
Do you plan to update the Spruce settings yourself shortly, or are you planning to ask a UI engineer to do this? I think this should be relatively simple to add, and merge these together. Otherwise, I don't think we should document this until the feature is available.

Reply:
I created a matching Spruce PR [here](https://github.com/evergreen-ci/spruce/pull/2322)